modules/5_install/templates/install_vars.yaml

```diff
@@ -52,3 +52,29 @@ bastion_vip: "${bastion_vip}"
 
 cni_network_provider: ${cni_network_provider}
 cni_network_mtu: ${cni_network_mtu}
+
+%{ if luks_compliant && length(luks_config) > 0 ~}
+luks:
+ enabled: true
+ config:
+%{ for item in luks_config ~}
+    - thumbprint: ${item.thumbprint}
+      url: ${item.url}
+%{ endfor ~}
+ filesystem_device: ${luks_filesystem_device}
+ format: ${luks_format}
+ wipeFileSystem: "${luks_wipeFileSystem}"
+ device: ${luks_device}
+ label: ${luks_label}
+%{ if luks_options != "" }
+ options:
+%{ for item in luks_options ~}
+    - ${item}
+%{ endfor ~}
+%{ endif ~}
+ wipeVolume: "${luks_wipeVolume}"
+ name: ${luks_name}
+%{ else ~}
+luks:
+ enabled: false
```

Comment:
No need of this else block.

Reply:
By default luks.enable should be false.

Reply:
Removed **else condition** from above code as per your suggestion as we are adding default value  luks.enabled to false in playbook as default value  mentioned below right?

ocp4-playbooks/playbooks/roles/ocp-config/defaults/main/main.yaml

```
luks:
   enabled: false
   filesystem_device: /dev/mapper/root
   format: xfs
   wipeFileSystem: true
   device: /dev/disk/by-partlabel/root
   label: luks-root
   options:
      - --cipher
      - aes-cbc-essiv:sha256
   wipeVolume: true
   name: root
```